### DISCUSSION TITLE
Ability to choose quality of photos backed up

### DISCUSSION THREAD

**Discussion by ua741** [ente-io/ente#951] [Enhancements]
By default photos should be backed up in their original quality. But I should have an option to lower the quality to conserve space.
[Roadmap 57 Votes](https://roadmap.ente.io/ability-to-choose-quality-of-photos-backed-up-p-1150/)

<img width="809" alt="image" src="https://github.com/ente-io/photos-app/assets/254676/a757b0e7-1a03-4ab2-be47-2b73c122647a">

**Comment by carobinario** [comment]
Hi, any update about this feature? Google Photos app has it and I like it because it helps user to save space and...time.

Thanks.
